.ci/release.sh

```diff
@@ -104,14 +107,26 @@ export COMMON_GRADLE_DEPLOY_PARAMS="$COMMON_GRADLE_SIGNING_PARAMS $COMMON_GRADLE
 
 if [[ "$target_specifier" == "all" ||  "$target_specifier" == "mavenCentral" ]]; then
   echo "--- Release the binaries to Maven Central"
-  ./gradlew publishElasticPublicationToSonatypeRepository closeAndReleaseSonatypeStagingRepository $COMMON_GRADLE_DEPLOY_PARAMS
+  if [[ "$dry_run" == "true" ]] ; then
+    echo './gradlew publishElasticPublicationToSonatypeRepository closeAndReleaseSonatypeStagingRepository'
```

Comment:
Just so you know (probably it's not an issue for the purposes of this "dry-run" feature) but without the signing key parameters this command will fail

Reply:
I didn't want to echo any potential variable which might be interpolated with some sensitive data. I've just found that the only way to mask things with Buildkite is by using some pre-scripts that need to be stored somewhere else in the repository.

Reply:
Oh! sorry I didn't realize it's just printing it, not actually running the command 😅